.env.test

```diff
@@ -11,3 +11,6 @@ EXPORT_DATE_OVERRIDE=
 EPOCH_TIMESTAMP=1568059200
 
 ROLLBAR_TOKEN=
+
+AURORA_BOOTSTRAP_ROLLBAR_TOKEN_FILE=rollerbar_token
+AURORA_BOOTSTRAP_DB_CRED_FILE=db_pass
```

Comment:
I really like the idea of prefixing env vars with `AURORA_BOOTSTRAP_` (or `AB_`)! However, can we commit to either prefixing all of them, or do it later? Otherwise it's unclear whether these are the only ab vars, or of there are others.

Reply:
Remove the prefix for simplicity -- saving typing efforts